getting-started/configure-datastreams.md

```diff
@@ -19,24 +19,24 @@ A datastream is a configured instance of the Edge Network that contains settings
 The following steps illustrate how to create a sample datastream for a mobile implementation:
 
 1. In the browser, navigate to [Adobe Experience Platform Launch](https://experience.adobe.com/launch) and login with your credentials.
-2. From the left panel, choose **Datatstreams** from the dropdown \(instead of **Client Side**\).
-3. Click **New Datatstream** \(located on top right\)
+2. From the left panel, choose **Datastreams** from the dropdown (instead of **Client Side**).
+3. Click **New Datastream** (located on top right)
 4. Set a name for the configuration and click **Save**.
 
-![Creating an Datastream in Adobe Experience Platform Launch](../.gitbook/assets/screen-shot-2021-02-02-at-12.42.57-pm%20%281%29.png)
+![Creating a datastream in Adobe Experience Platform Launch](../.gitbook/assets/screen-shot-2021-02-02-at-12.42.57-pm%20%281%29.png)
```

Comment:
Can you also update the screenshot to match the new Launch UI and the description?

Reply:
Hmm, I don't think I have access to the Launch UI, unfortunately - if you know someone who can take the screenshot for me, I'd be happy to update it!

Reply:
👍  provided couple screenshots to you